src/ParallelMultiplication/ParallelMultiplication/FileWorker.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.IO;
+
+namespace ParallelMultiplication
+{
+    /// <summary>
+    /// Provides read and write file function.
+    /// </summary>
+    public class FileWorker
```

Comment:
```suggestion
    public static class FileWorker
```
Иначе кто-нибудь будет создавать объекты этого класса или ещё что-то такое